runtime/allocator.cpp

```diff
@@ -306,6 +282,48 @@ MemoryReplacementGuard::~MemoryReplacementGuard() {
 
 } // namespace dl
 
+// sanitizers aren't happy with custom realization of malloc-like functions
+#if !ASAN_ENABLED and !defined(__APPLE__)
+
+extern "C" void *__libc_malloc(size_t size);
+extern "C" void *__libc_calloc(size_t nmemb, size_t size);
+extern "C" void *__libc_realloc(void *mem, std::size_t new_size);
```

Comment:
Let's make it consistent with other declarations:
```suggestion
extern "C" void *__libc_realloc(void *mem, size_t new_size);
```
Or maybe it's better to add ’std::’ in other declarations

Reply:
true, prefix removed